installing/installing_openstack/installing-openstack-installer-custom.adoc

```diff
@@ -14,7 +14,8 @@ In {product-title} version {product-version}, you can install a customized clust
 xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
 processes.
 
-* Have access to a {rh-openstack} administrator's account
+* Have a storage service installed in {rh-openstack}, like Block Storage (Cinder) or Object Storage (Swift)
+** Object storage is the recommended storage technology for {product-title} registry cluster deployment. For more information, see "Optimizing storage" in the "Scalability and performance" section of the documentation.
```

Comment:
Do you need a sub-bullet here? Could it be all in one bullet?

Also, could there be a link to Optimizing Storage here?

Reply:
Fair.